Replicator/Puller.cc

```diff
@@ -280,17 +283,17 @@ namespace litecore { namespace repl {
 
 
     void Puller::revReRequested(fleece::Retained<IncomingRev> inc) {
-        enqueue(FUNCTION_TO_QUEUE(Puller::_revReRequested), inc);
+        enqueue(FUNCTION_TO_QUEUE(Puller::_revReRequested), _missingSequences.bodySizeOfSequence(inc->remoteSequence()));
```

Comment:
You can't access `_missingSequences` in a public method since it's not necessarily called on the Puller's queue. Also, these public methods should be simple wrappers around `enqueue`, without any extra logic.

(We already talked about this on Slack, I'm just putting it here for completeness.)